### DISCUSSION TITLE
Command ExternalBuildToolExecution failed with a nonzero exit code

### DISCUSSION THREAD

**Discussion by travisdesell** [microsoft/AirSim#4759] [Support Q&A]
Hi,

We have been trying to compile and run AirSim on OSX (non-M1 macs) and are getting the following (not very useful) error:

Command ExternalBuildToolExecution failed with a nonzero exit code

Everything up to building in xcode works without any errors. We've tried cleaning, rebuilding and the error still happens. This is on OSX Montery (12.6) and Xcode 14.1

thanks!
-Travis

**Comment by Dre1896** [comment]
Good evening,

I've followed the instructions to installing Airsim on Mac using [this](https://microsoft.github.io/AirSim/build_macos/) guide and I'm stuck on the final portion of the installation guide "How to Use Airsim" --> "Open the XCode workspace, and press the Build and run button in the top left."

I have this error: 

![image](https://user-images.githubusercontent.com/6758438/208326810-a7064e5c-9c9c-432d-b83e-611df9fdad80.png)

I've done troubleshooting and tried to build the xcode but to no avail. This is on a MacBook Pro 2019 with a **Processor: 2.3 GHz 8-Core Intel Core i0**, **macOS Ventura Version 13.0**, and **XCode version 14.2**
